scanner/src/routes/+page.svelte

```diff
@@ -192,13 +200,49 @@
 </script>
 
 <main>
-  <div id="reader" />
-  {#if !scanning}
-    <div class="controls">
-      <button on:click={startScan}>Start scanning</button>
-      <button on:click={scanPreset}>Simulate scan of preset data</button>
+  <div id="reader">
+    {#if !scanning}
+      <div class="background" />
+    {/if}
+    <div class="relative barcode-scanner-area-wrapper">
+      <div class="relative barcode-scanner-area">
+        <img class="subtract-image" src={subtract} alt="subtract " />
+        {#if scanning}
+          <div class="square surround-cover" />
+        {:else}
+          <img class="logo" src={logo} alt="logo" />
+        {/if}
+      </div>
+      <div class="relative">
+        {#if scanning}
+          <p class="description">Scan a Product Passport QR code to view product data</p>
+        {:else}
+          <p class="description">
+            Turn on your camera for scan a Product Passport QR code to view product data
+          </p>
+        {/if}
+      </div>
+      {#if !scanning}
+        <div class="relative button-wrapper">
+          <Button onClick={startScan} icon={camera} title="Turn on" />
```

Comment:
For some reason this button doesn't behave like a button, it should have a :point_up_2: cursor when hovering.

![Peek 2023-09-22 10-35](https://github.com/ioxiocom/ioxio-tags/assets/1296558/f8cf6047-7ee5-4806-8889-1213563e189d)